lua/lvim/core/autocmds.lua

```diff
@@ -99,6 +99,28 @@ function M.load_defaults()
         end,
       },
     },
+    {
+      "ColorScheme",
+      {
+        group = "_lvim_colorscheme",
+        callback = function()
```

Comment:
since there's no `ColorSchemePost` we should probably defer this function just to be safe

```suggestion
        callback = function(vim.schedule(function()
```